proto/umee/incentive/v1/query.proto

```diff
@@ -95,6 +95,7 @@ message QueryPendingRewardsResponse {
 // QueryBonded defines the request structure for the Bonded gRPC service handler.
 message QueryBonded {
   string address = 1;
+  string denom   = 2;
```

Comment:
shouldn't this be optional? if yes, then mark it optional and update the comment.

Reply:
In protobuf, `optional` is incompatible with `string` (but the string can be empty).